src/emitter.js

```diff
@@ -41,8 +42,10 @@ export default class UniversalEmitter {
     if (index > -1) this.handlers[type].splice(index, 1);
   }
   emit(type, payload, ctx) {
-    const mappers = this.mappers[type] || [];
-    const handlers = this.handlers[type] || [];
+    const globalMappers = this.mappers[globalEventType] || [];
+    const globalHandlers = this.handlers[globalEventType] || [];
+    const mappers = (this.mappers[type] || []).concat(globalMappers);
+    const handlers = (this.handlers[type] || []).concat(globalHandlers);
     const event = {
       type,
       payload: mappers.reduce((payload, mapper) => {
```

Comment:
It looks like we previously had an implicit wildcard match in `getArgs` below, which mapped to all key-less types.  Given this change, that would no longer be supported (a la 'all key-less types are handled together').  Is that something we wanted to support?

Reply:
I think the intended behavior of that was to have * behave as a wildcard. I don't think there is any value in supporting events with no type.